ocdskit/oc4ids.py

```diff
@@ -29,7 +30,7 @@ def cast_number_or_zero(item):
     """ Cast to decimal if fail return 0 so summing still works."""
     try:
         return decimal.Decimal(item)
-    except ValueError:
+    except (ValueError, TypeError):
```

Comment:
In what case is TypeError raised?

Reply:
When decimal.Decimal(None) it raises a type error.  I was expecting a value error but this is not the case.